CMakeLists.txt

```diff
@@ -103,6 +103,7 @@ option(ACTS_BUILD_EXAMPLES_HEPMC3 "Build HepMC3-based code in the examples" OFF)
 option(ACTS_BUILD_EXAMPLES_HASHING "Build Hashing-based code in the examples" OFF)
 option(ACTS_BUILD_EXAMPLES_PYTHIA8 "Build Pythia8-based code in the examples" OFF)
 option(ACTS_BUILD_EXAMPLES_PYTHON_BINDINGS "Build python bindings for the examples" OFF)
+option(ACTS_BUILD_EXAMPLES_ROOT "Build modules based on ROOT I/O" OFF)
```

Comment:
I saw that you had a discussion with the Robot already about this, but I'm wondering if we shouldn't turn this on by default actually, since it aligns with the *previous* configuration.

Reply:
Can do it, I don't have a strong opinion on this...